aums/attendance.html

```diff
@@ -42,10 +42,10 @@
             <li class="nav-item">
                 <a aria-disabled="true" class="nav-link" href="http://rajkumaar.co.in" tabindex="-1">About</a>
             </li>
-            <li class="nav-item">
+            <!--<li class="nav-item">
                 <a class="nav-link" onclick="localStorage.removeItem(" sem-for-atd");localStorage.removeItem("sem-for-grades");alert("Cache
                 cleared");">Clear Cache</a>
-            </li>
+            </li>-->
```

Comment:
It is not a recommended practice to keep commented code. Since git anyways maintains the history, it's not a big deal to delete these lines.